resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,279 @@
+        primaryLabel: 'Genre',
+        altLabels: ['Subgenre'],
+        fallbackValue: game.genre,
+      }),
+    [allGameHubs, game.genre],
+  );
+
+  const languageRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Meta - Language -', 'Meta - Language Patch -'],
+        primaryLabel: 'Meta - Language',
+        altLabels: ['Meta - Language Patch'],
+        altLabelsLast: true,
+        markAltLabels: true,
+      }),
+    [allGameHubs],
+  );
+
+  const themeRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Theme -'],
+        primaryLabel: 'Theme',
+      }),
+    [allGameHubs],
+  );
+
+  const perspectiveRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Perspective -'],
+        primaryLabel: 'Perspective',
+      }),
+    [allGameHubs],
+  );
+
+  const featureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Feature -', 'Game Mechanic -'],
+        primaryLabel: 'Feature',
+        altLabels: ['Game Mechanic'],
+        altLabelsLast: false,
+      }),
+    [allGameHubs],
+  );
+
+  const creditRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Credits -'],
+        primaryLabel: 'Credits',
+      }),
+    [allGameHubs],
+  );
+
+  const technicalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Technical -'],
+        primaryLabel: 'Technical',
+      }),
+    [allGameHubs],
+  );
+
+  const miscRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Misc. -'],
+        primaryLabel: 'Misc.',
+      }),
+    [allGameHubs],
+  );
+
+  const protagonistRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Protagonist -'],
+        primaryLabel: 'Protagonist',
+      }),
+    [allGameHubs],
+  );
+
+  const settingRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Setting -'],
+        primaryLabel: 'Setting',
+      }),
+    [allGameHubs],
+  );
+
+  const regionalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Regional -'],
+        primaryLabel: 'Regional',
+      }),
+    [allGameHubs],
+  );
+
+  const raFeatureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: [
+          'Meta -',
+          'Rollout Sets -',
+          'AotW -',
+          'RANews -',
+          'RA Awards -',
+          'Custom Awards -',
+          'Console Wars',
```

Comment:
This doesn't seem to be matching "Console Wars I": http://localhost:64000/game2/24227

![image](https://github.com/user-attachments/assets/99695f45-77c1-45c3-b782-71307a6ebe59)

Reply:
These are now captured in the "Events" row: https://github.com/RetroAchievements/RAWeb/pull/3522#discussion_r2094625382